modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/util/OpenAPIDeserializerTest.java

```diff
@@ -49,6 +49,47 @@
 public class OpenAPIDeserializerTest {
 
     @Test
+    public void securityTest(@Injectable List<AuthorizationValue> auths){
+        String yaml = "openapi: \"3.0.0\"\n" +
+                "info:\n" +
+                "  title: Test OpenApi Specification\n" +
+                "  version: 1.0.0\n" +
+                "paths:\n" +
+                "  /test:\n" +
+                "    get:\n" +
+                "      operationId: testGet\n" +
+                "      security:\n" +
+                "      - openId:\n" +
+                "        - https://scopes.example.com/myScope\n" +
```

Comment:
Great, just one last question. Why not updated "https://scopes.example.com/myScope" to reflect real world scope example. :)